compiler-rt/lib/builtins/cpu_model/cpu_model.h

```diff
@@ -31,7 +31,15 @@
 // We're choosing init priority 90 to force our constructors to run before any
 // constructors in the end user application (starting at priority 101). This
 // value matches the libgcc choice for the same functions.
-#define CONSTRUCTOR_ATTRIBUTE __attribute__((constructor(90)))
+#ifdef _WIN64
```

Comment:
Wouldn't it be more correct and generic to make this `_WIN32` (which applies to both 32 and 64 bit windows)? I get that the current change is for aarch64 only, but this does seem to be a quite architecture independent file, and unless the distinction between 32 and 64 is needed (which it seldom is) I prefer using `_WIN32` as define to look for.

Reply:
ack.

Reply:
This comment still seems to be unresolved.

Reply:
ah sorry, fixed in [eda3898](https://github.com/llvm/llvm-project/pull/111961/commits/eda389814bc59c5d5063d25be526fc223f3ecc34).